app/Http/Controllers/Api/ApplicationController.php

```diff
@@ -168,7 +168,7 @@ public function updateReview(Request $request, JobApplication $application)
             $this->setAvailability($application);
         }
 
-        $review->fresh();
+        $review = $review->fresh();
```

Comment:
Just so you know, you can also use `->refresh()` to refresh the object in place.